worker/prices_test.go

```diff
@@ -0,0 +1,97 @@
+package worker
+
+import (
+	"context"
+	"errors"
+	"testing"
+	"time"
+
+	rhpv4 "go.sia.tech/core/rhp/v4"
+	"go.sia.tech/core/types"
+	"go.sia.tech/renterd/api"
+	"go.sia.tech/renterd/internal/test/mocks"
+)
+
+func TestPricesCache(t *testing.T) {
```

Comment:
should probably be extended to cover the oversight on `cost`

Reply:
There is actually no cost in rhp4. Returning it was just a leftover